### ISSUE TITLE
Implement API awards end point

### ISSUE THREAD

**Issue by lane55** [pc2ccs/pc2v9#562]
**Is your feature request related to a problem?** 

Yes, other tools use the awards/ endpoint.

**Feature Description**: 

Add awards endpoint to API

**Have you considered other ways to accomplish the same thing?** 

**Do you have any specific suggestions for how your feature would be ***implemented*** in PC^2?**

**Additional context**:

**Comment by lane55** [comment]
Add high priority so issue can be considered to be finished soon, rather than someday.

**Comment by lane55** [comment]
Likely depends on #383 for the source of the awards json/classes.

**Comment by lane55** [comment]
One bug FTS team number incorrect.   The citations may not match because of a misunderstanding
of what data is in ciation.
first-to-solve-catering NOT_SAME citation 'First to solve problem catering' vs 'First to solve problem A' 
first-to-solve-catering NOT_SAME team number '156' vs '35' 
first-to-solve-catering2 NOT_SAME citation 'First to solve problem catering2' vs 'First to solve problem F'

**Comment by lane55** [comment]
group winner also incorrect
group-winner-17124 NOT_SAME citation 'Winner(s) of group UNSW Sydney' vs 'Winner(s) of group South Pacific' 
group-winner-17124 NOT_SAME team number '131' vs '120'

**Comment by lane55** [comment]
The comparison of the awards is a tool/class from #592

**Comment by clevengr** [comment]
This seems to be a duplicate of Issue #787, although that issue has documentation regarding the issue that's not present in this issue -- so it probably makes sense to retain both issues until a PR is created; at that point the PR should close BOTH #562 and #787.
